tests/commands/update/test_update_panelapp_green.py

```diff
@@ -32,42 +32,46 @@ def test_update_panelapp_green_non_existing_institute(empty_mock_app):
 
 
 @responses.activate
-def test_update_panelapp_green(mock_app):
+def test_update_green_panel(mock_app):
     """Tests the CLI that creates/updates PANELAPP-GREEN panel in database"""
 
     # GIVEN that no PANELAPP-GREEN panel exists in database:
     assert store.gene_panel(panel_id=PANEL_NAME) is None
 
     # GIVEN a mocked response from PanelApp list_panels endpoint
+    with open(panelapp_panels_reduced_path) as f:
+        panels_data = json.load(f)
+
     responses.add(
         responses.GET,
-        PANELAPP_LIST_PANELS_URL,
-        json={"result": [{"Panel_Id": DEMO_PANEL}]},
+        API_PANELS_URL,
+        json=panels_data,
+        match=[responses.matchers.header_matcher({"Content-Type": "application/json"})],
         status=200,
     )
 
     # GIVEN a mocked response from PanelApp get_panel endpoint (mock response returns test PanelApp panel from scout/demo folder)
-    data = None
     with open(panelapp_panel_path) as f:
-        data = json.load(f)
+        panel_data = json.load(f)
     responses.add(
         responses.GET,
         PANELAPP_GET_PANEL_URL,
-        json=data,
+        json=panel_data,
+        match=[responses.matchers.header_matcher({"Content-Type": "application/json"})],
         status=200,
     )
 
-    # GIVEN an app CLI
+    # GIVEN a cli runner
     runner = mock_app.test_cli_runner()
+    assert runner
 
     # THEN command to create the PANELAPP-GREEN panel should be successful
-    result = runner.invoke(cli, ["update", "panelapp-green", "-i", "cust000"])
+    result = runner.invoke(cli, ["update", "panelapp-green", "-i", "cust000"], input="1")
```

Comment:
Ah, cool - I didn't know that one! Still, I think a named option for the panel types would not hurt!

Reply:
Took me a while to remember how it was done, but we do have the same thing also in other tests:

<img width="900" alt="image" src="https://github.com/user-attachments/assets/11a82350-cd41-4d90-a44f-ed50693d345f">

Reply:
Right, I do vaguely remember now. 😁 But nice anyway!